samples/dashboards/demos/custom-component/demo.js

```diff
@@ -1,8 +1,8 @@
 const { ComponentRegistry, Component } = Dashboards;
 
 class YouTubeComponent extends Component {
-    constructor(options) {
-        super(options);
+    constructor(cell, options) {
+        super(cell, options);
```

Comment:
You should update docs too.

https://github.com/highcharts/highcharts/blob/db/asynchronous-component-update/docs/dashboards/custom-component.md